src/model.ts

```diff
@@ -1013,47 +1029,35 @@ export class HeatmapcustomColumn extends ValueColumn<number[]> {
     const b1_q3 = getPercentile(b_val, 75);
 
 
-    if (this.desc['sort'] == 'min') {
+    if (this.desc.sort === 'min') {
```

Comment:
I would store the current sort criteria as property of the model class not part of the description. The description should be considered readonly and just used as input data.